back/api/models.py

```diff
@@ -463,6 +463,12 @@ class Meta:
         db_table = 'order_item'
         verbose_name = _('order_item')
 
+    @property
+    def available_formats(self):
+        queryset = ProductFormat.objects.filter(
+            product=self.product).values_list('data_format__name', flat=True)
```

Comment:
`flat=True`: this is done on purpose, right? You want a raw value, not the Id of this one?

Reply:
Yes, `values_list` returns a list of tuples by default, `flat=True` returns a list of string instead of the half empty tuple: `('DXF',),('ITF',)`